sql/src/test/java/org/apache/druid/sql/calcite/schema/SystemSchemaTest.java

```diff
@@ -1448,6 +1463,129 @@ public void testSupervisorTableAuth()
     // verifyTypes(rows, SystemSchema.SUPERVISOR_SIGNATURE);
   }
 
+  @Test
+  public void testPropertiesTable() throws Exception
+  {
+    SystemPropertiesTable propertiesTable = EasyMock.createMockBuilder(SystemPropertiesTable.class)
+                                                    .withConstructor(druidNodeDiscoveryProvider, authMapper, httpClient, MAPPER)
+                                                    .createMock();
+
+    EasyMock.replay(propertiesTable);
+
+    final DruidNodeDiscovery coordinatorNodeDiscovery = EasyMock.createMock(DruidNodeDiscovery.class);
+    final DruidNodeDiscovery middleManagerNodeDiscovery = EasyMock.createMock(DruidNodeDiscovery.class);
+    final DruidNodeDiscovery brokerNodeDiscovery = EasyMock.createMock(DruidNodeDiscovery.class);
+    final DruidNodeDiscovery routerNodeDiscovery = EasyMock.createMock(DruidNodeDiscovery.class);
+    final DruidNodeDiscovery historicalNodeDiscovery = EasyMock.createMock(DruidNodeDiscovery.class);
+    final DruidNodeDiscovery overlordNodeDiscovery = EasyMock.createMock(DruidNodeDiscovery.class);
+    final DruidNodeDiscovery peonNodeDiscovery = EasyMock.createMock(DruidNodeDiscovery.class);
+    final DruidNodeDiscovery indexerNodeDiscovery = EasyMock.createMock(DruidNodeDiscovery.class);
+
+    EasyMock.expect(druidNodeDiscoveryProvider.getForNodeRole(NodeRole.COORDINATOR)).andReturn(coordinatorNodeDiscovery).once();
```

Comment:
Please group the definition and expectation of each mock together.
Also, since these snippets will follow the same format, please see if they can't be put in a common private method.

Something liek:

```
final DruidNodeDiscovery coordinatorNodeDiscovery = mockNodeDiscovery(
    // list of nodes to return
    new DruidNode(...),
    new DruidNode(...)
);

final DruidNodeDiscovery overlordNodeDiscovery = mockNodeDiscovery(
       new DruidNode(...),
    new DruidNode(...)
);
```

```
private DruidNodeDiscovery mockNodeDiscovery(DruidNode... nodes)
{
   // Create the mock
   // Set expectations
   // Replay 
}
```